src/main.rs

```diff
@@ -78,6 +78,10 @@ struct Cli {
     /// Ignore settings from .git-nbconfig.yaml
     ignore_git_nb_config: bool,
 
+    #[clap(long, action)]
+    /// Specify a regex to use to strip matching output
```

Comment:
This makes --help confusing. Can we explain a bit more of the order and precedence and maybe include an example in the README or --help of how one may want to use it (our example is removing closed widgets).